HydroModelBuilder/ModelInterface/flopyInterface/viz/MT3D_PP_viz.py

```diff
@@ -186,17 +188,26 @@ def viewConcsByZone(self, nper='all', specimen=None):
     modelmap.plot_ibound()
 
     modelmap.plot_bc('RIV', plotAll=True)
-    modelmap.plot_bc('WEL', plotAll=True)
+    try:
+        modelmap.plot_bc('WEL', plotAll=True)
+    except Exception:
+        pass
     modelmap.plot_bc('GHB', plotAll=True)
     modelmap.plot_bc('SFR', plotAll=True)
-    modelmap.plot_bc('DRN', plotAll=True)
+    try:
+        modelmap.plot_bc('DRN', plotAll=True)
+    except Exception:
+        pass
     ax.axes.xaxis.set_ticklabels([])
 
     ax = fig.add_subplot(2, 4, 2, aspect='equal')
     ax.set_title('Coonambidgal')
     modelmap = flopy.plot.ModelMap(model=self.mf_model.mf)
-    max_conc = -100.0
-    min_conc = 1E6
+    min_conc = -100.0
+    max_conc = 100.0
+    temp = max_conc
```

Comment:
This temp variable looks important but is never used.

Reply:
@ConnectedSystems , very important indeed.
Delete